hack/install-database.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+kubectl apply -f - <<EOF
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: databases
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: postgresql
+  name: postgresql
+  namespace: databases
+spec:
+  selector:
+    matchLabels:
+      app: postgresql
+  template:
+    metadata:
+      labels:
+        app: postgresql
+    spec:
+      containers:
+        - name: postgresql
+          image: postgres:16
+          ports:
+            - containerPort: 5432
+          env:
+            - name: POSTGRES_DB
+              value: root
+            - name: POSTGRES_USER
+              value: root
+            - name: POSTGRES_PASSWORD
+              value: password  # notsecret
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: postgresql
+  name: postgresql
+  namespace: databases
+spec:
+  type: ClusterIP
+  ports:
+    - port: 5432
+  selector:
+    app: postgresql
+
+EOF
+
+kubectl rollout status deployment --namespace=databases --timeout=30s
+sleep 15  # Wait for PostgreSQL to be really ready
```

Comment:
I'm not sure, but maybe 15 sec is not enough :(

Reply:
I tested this multiple times and it is, its not enough when you tested this PR?

Reply:
I've just tested. My comment was based on my previous tests, when I had to wait more than 15 sec to have the DB working. 
My results from testing the script are that the 15 sec are enough. I tested also deploying the DB after delete the postgresql:16 image, to force the pull.